src/einsteinpy/plotting/geodesics_scatter.py

```diff
@@ -9,16 +9,28 @@
 class ScatterGeodesicPlotter:
     """
     Class for plotting static matplotlib plots.
+
+    Parameters
```

Comment:
Again, why does the class docstring contains this?

Reply:
I have shifted them to the __init__ docstring. If this is also wrong. Then it would be really helpful to know where to put them.

Reply:
Also, there was no docstring for plot_trajectory so I have added them. Do let me know if it was unnecessary.